gulpfile.js

```diff
@@ -173,7 +173,10 @@ declareExtensionVersionAlias(
  *   name: ?string,
  *   version: ?string,
  *   hasCss: ?boolean,
- *   loadPriority: ?string
+ *   loadPriority: ?string,
+ *   cssBinaries: ?Array<string>,
+ *   extraGlobs?Array<string>,
+ *   bundleOnlyIfListedInFiles: ?boolean
```

Comment:
This is a declaration for the same key that's currently being used in `buildExtensions`, correct?

Reply:
i think so. i was having a hard time tracking if its still the same options object but i believe so

Reply:
I suppose testing with `--files` should make it clear.

Reply:
Also, it's worth testing this code with a couple variations of `gulp | gulp dist --fortesting` with the args `--noextensions | --extensions=minimal_set` since that behavior depends on CSS compilation.

Reply:
done. tested